Userland/Libraries/LibWeb/SVG/SVGSymbolElement.cpp

```diff
@@ -30,6 +30,8 @@ void SVGSymbolElement::initialize(JS::Realm& realm)
 // https://svgwg.org/svg2-draft/struct.html#SymbolNotes
 void SVGSymbolElement::apply_presentational_hints(CSS::StyleProperties& style) const
 {
+    Base::apply_presentational_hints(style);
```

Comment:
This is not making properties inheritable (which is a CSS concept), but rather adding a missing call to the `SVGGraphicsElement` base class which ensures that `SVGSymbolElement` supports all the presentation attributes it's supposed to.

Reply:
The patch looks great though, just the commit message is misleading :)

Reply:
Also, can we add a test for this? If a layout test is not possible, a ref test should still work :)

Reply:
Is this better?

Reply:
I've removed a commit by mistake and the PR has closed itself. Should I make a new one or a reopening is possible?